tests/context.py

```
@@ -1001,9 +1002,11 @@ def managed_process(process):
 				if process.is_alive():
 					process.kill()
 		except Exception as e:
-			if (sys.stderr.isatty()):
+			if (__debug__ and sys.stderr.isatty()):
 				# Log the error but don't re-raise as this is cleanup code
-				print(f"Error during process cleanup: {e}", file=sys.stderr)
+				warnings.warn(
+					f"Error during process cleanup: {e}", stacklevel=2
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)